test/runnable/test18076.sh

```diff
@@ -4,6 +4,6 @@ output_file=${OUTPUT_BASE}.log
 
 echo 'import std.stdio; void main() { writeln("Success"); }' | \
 	$DMD -m${MODEL} -run - > ${output_file}
-grep -q '^Success$' ${output_file}
+grep -q 'Success' ${output_file}
```

Comment:
Is this safe in case there is a compilation error that includes the string "Success" from the code? If errors are written to stderr, then it should be ok.

Reply:
Errors are written to stderr.